plugins/qtAccessionPlugin/modules/accession/templates/_alternativeIdentifiersIndex.php

```diff
@@ -0,0 +1,14 @@
+<div class="field">
+  
+  <h3><?php echo __('Alternative identifier(s)') ?></h3>
```

Comment:
We may want to review the CAAIS wording around the identifier field so that AtoM's WebUI is close. http://archivescanada.ca/CWG_AccessionStandard#1.2_e

Maybe just "Identifier(s)" here?